### PR TITLE
Add Graphics tests for handles and construction

### PR DESCRIPTION
**Pull request by hughbe**
Let's see how the CI fares as some of these tests use HWNDs

### CODE REVIEW

File: src/System.Drawing.Common/src/System/Drawing/Graphics.cs [hunk 1]
```diff
@@ -233,17 +229,7 @@ public static Graphics FromImage(Image image)
...
 
 
-        /// <devdoc>
-        ///     Gets the GDI+ native graphics pointer.
-        /// </devdoc>
-        internal IntPtr NativeGraphics
-        {
-            get
-            {
-                Debug.Assert(_nativeGraphics != IntPtr.Zero, "NativeGraphics == IntPtr.Zero.");
```
Comment (stephentoub) [inline src/System.Drawing.Common/src/System/Drawing/Graphics.cs]: Why is this assert invalid?  There are code paths that successfully hit this even when _nativeGraphics hasn't been initialized?

File: src/System.Drawing.Common/tests/GraphicsTests.cs [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+            {
+                IntPtr hdc = graphics.GetHdc();
+                Assert.NotEqual(IntPtr.Zero, hdc);
+
+                using (Graphics graphicsCopy = Graphics.FromHdc(hdc))
+                {
+                    VerifyGraphics(graphicsCopy, graphicsCopy.VisibleClipBounds);
+                }
+
+                graphics.ReleaseHdc();
```
Comment (stephentoub) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: Will anything bad happen if VerifyGraphics asserts and this release line is never executed?

File: src/System.Drawing.Common/src/System/Drawing/Graphics.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Drawing.Common/src/System/Drawing/Graphics.cs]: This assertion is hit in the test `GetHdc_Disposed_ThrowsObjectDisposedException`

File: src/System.Drawing.Common/tests/GraphicsTests.cs (hunk 2 above)
Comment (hughbe) [inline src/System.Drawing.Common/tests/GraphicsTests.cs]: Worse case scenario is we'd get a memory leak. But it also looks like the finalizer properly releases the HDC if it still exists

File: src/System.Drawing.Common/src/System/Drawing/Graphics.cs (hunk 1 above)
Comment (hughbe) [inline src/System.Drawing.Common/src/System/Drawing/Graphics.cs]: ... and added more tests that hit this assertion

**Comment by stephentoub** [comment]
Thanks, Hugh.  Can you rebase?  There's a conflict with one of your other PRs that was merged.

**Comment by hughbe** [comment]
Rebased, sorry

**Comment by hughbe** [comment]
Test Linux x64 Release Build

**Pull request merged**